.replit

```diff
@@ -0,0 +1,2 @@
+language = "python3"
+run = "/ultimatepython"
```

Comment:
Add a newline to this `.replit` file as this is best practice.

Also change the run command to `run = "python runner.py"` as "/ultimatepython" is not a valid command.

Reply:
Actually this change is simple enough that I can do it myself 😄

More of a FYI